cypress/commons/actions/generic/DatasetManager.js

```diff
@@ -8,7 +8,11 @@ const SELECTORS = GENERIC_SELECTORS.datasetmanager;
 
 export const getDatasetManagerTab = (timeout = 5) => cy.get(SELECTORS.tabName, { timeout: timeout * 1000 });
 export const getDatasetManagerView = (timeout = 5) => cy.get(SELECTORS.view, { timeout: timeout * 1000 });
-export const switchToDatasetManagerView = () => getDatasetManagerTab().click();
+export const switchToDatasetManagerView = (queries = []) => {
+  const alias = api.interceptPostDatasetTwingraphQueries(queries, null, queries?.length ?? 1);
+  getDatasetManagerTab().click();
+  if (Array.isArray(queries) && queries?.length > 0) api.waitAliases(queries.map(() => alias));
```

Comment:
The logic of handling the value of `queries` should probably be "hidden" in `interceptPostDatasetTwingraphQueries` instead of being implemented here  in `switchToDatasetManagerView`.

Could we make `interceptPostDatasetTwingraphQueries` return an array of aliases, and simply call `api.waitAliases` on this array? (if the array is empty, `api.waitAliases` will just ignore it and do nothing)

Also, I'm not sure how cypress will act when waiting multiple times for the same alias, but I think it's equivalent to waiting for it once. Do we really need to wait multiple aliases? (idem in `selectDatasetById` and `openCategoryDetailsDialog`)